lib/ansible/plugins/callback/profile_tasks.py

```diff
@@ -133,7 +133,7 @@ def playbook_on_stats(self, stats):
 
         # Print the timings
         for uuid, result in results:
-            msg=u"{0:-<70}{1:->9}".format(result['name'],u' {0:.02f}s'.format(result['time']))
+            msg=u"{0:-<70} {1:->9}".format(result['name'],u' {0:.02f}s'.format(result['time']))
```

Comment:
It doesn't solve the spacing/padding problem, now we have this (one space at 9 columns before the end, i.e after the right-padding of the name and before the left-padding of the time):

```
before ------------------------------------------------------------------ 0.00s
after----------------------------------------------------------------- --- 0.00s
```